Update myApps info for Stanage go live. (#1706)
rcgsheffield/sheffield_hpc: hpc/connecting.rst

Connecting to a cluster using myApps (web browser)
==================================================

For easier access to HPC resources,  IT Services runs an instance of Oracle Secure Global desktop called myApps to provide web based access to CLI terminals,
text editors within interactive sessions for the ShARC, Bessemer and Training HPC clusters.

.. important:: 
    

>     Access to the Stanage cluster via the myApps service is currently unavailable.

Reviewer: ```suggestion
    The myApps service is only accessible on Bessmer and ShARC (_not_ Stanage).
```
as the current proposed wording implies could be available on Stanage shortly.

Reviewer: Nope, looks good

> In order to access the HPC clusters you must set up a `VPN and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_. 

Reviewer: ```suggestion
In order to access the HPC clusters you must set up a `VPN connection and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_. 
```

Also see section **Whether/how you can connect** below. 

The web browser method of access to ShARC and Bessemer is recommended. This method works well for most browsers on all the common 
computing platforms (Linux, Windows, Mac), however we recommend Chrome or Firefox.

> :underline-bold:`How to login to the myApps service`

Reviewer: Better to use (sub)section heading here? https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections
